scripts/steamcmd_installer.sh

```diff
@@ -2,7 +2,7 @@
 # steamcmd Base Installation Script
 #
 # Server Files: /mnt/server
-# Image to install with is 'debian:buster-slim'
+# Image to install with is 'ghcr.io/parkervcp/games:source'
```

Comment:
This is not an install image

Reply:
fhfhfhfhfhfhfhhfhfhhffhhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhfhffhhfhf